bot.py

```diff
@@ -164,5 +164,19 @@ async def on_command_error(ctx, error):
     if isinstance(error, commands.errors.CheckFailure):
         await ctx.send("You do not have the correct role for this command.")
 
+    # Send an error message when the user input invalid command
+    elif isinstance(error, commands.CommandNotFound):
+        error = str(error)
+        error = error[error.find('"') + 1 : error.rfind('"')]
```

Comment:
what does this do?

Reply:
![Screen Shot 2021-07-04 at 2 04 48 AM](https://user-images.githubusercontent.com/47313362/124374815-465cd600-dc6c-11eb-98b6-813a4afbca93.png)
when the user inputs an invalid command, the error sends the message like the above screenshot.
So that is why I changed the error message into string type and split it to get the command what the user inputs.

BUT, I do not need these lines because you said ```remove first command.
eg; aeorijaer is not a valid command``` in next review

Reply:
huh? my comment was to remove `**command** is not a valid command`

Reply:
If not found, just print that discord error message out. 
I don't think we need to format it again

Reply:
ohhhh, I see,, I will fix it righ away

Reply:
<img width="403" alt="Screen Shot 2021-07-04 at 2 37 56 AM" src="https://user-images.githubusercontent.com/47313362/124375556-d7ce4700-dc70-11eb-8a43-6c780e790e54.png">

This is the output using discord error message